packages/hardhat-core/src/internal/hardhat-network/provider/transactions/ReadOnlyValidEIP1559Transaction.ts

```diff
@@ -57,7 +57,7 @@ export class ReadOnlyValidEIP1559Transaction extends FeeMarketEIP1559Transaction
   constructor(sender: Address, data: FeeMarketEIP1559TxData = {}) {
     const fakeCommon = new Common({
       chain: "mainnet",
-      hardfork: "london",
+      hardfork: "london", // TODO: consider chaning this to ["latest hardfork"]
```

Comment:
Same

Reply:
On a call, Pato said this is fine as-is, TODO and all.